content/kubeone/master/using_kubeone/cluster_reconciliation.en.md

```diff
@@ -42,7 +42,7 @@ terminal and ability to confirm actions, you can use the `--auto-approve`
 flag:
 
 ```bash
-kubeone apply --manifest config.yaml -t . --auto-approve
+kubeone apply --manifest config.yaml -t tf.json --auto-approve
```

Comment:
where did `tf.json` came from?

Reply:
That’s how we usually call it in all the other documents.

Reply:
I meant, in here it's not clear (from previous text) from where this file popup-ed

Reply:
Fixed.